frontend/src/components/V4Select.vue

```diff
@@ -214,12 +214,12 @@ button.v4-select {
   height:100%;
   width:100%;
 }
-button.v4-select:focus{
-   box-shadow: 0 0 0 3px rgba(21, 156, 228, 0.4);
-   outline:none;
+button.v4-select:hover {
+
+}
+button.v4-select:focus {
```

Comment:
not sure how to handle this, but the V4 select is used in the 3rd tab of the search response. When using  keyboard to navigate through them, the first 2 focus very differently from the 3rd.

Reply:
also noticed that the caret that is in the dropdown is white. When focused, it gets lost in the light blue highlight

Reply:
It's because it's a dropdown - I'm counting on the color consistency to carry over the pattern. Once this is on dev, I'll pass it by Jack and Melinda for their expertise.

Reply:
but I totally get it - it's an odd control

Reply:
er... "non-standard", not odd